src/main/java/com/mercadopago/example/apis/order/ProcessOrderById.java

```diff
@@ -0,0 +1,33 @@
+package com.mercadopago.example.apis.order;
+
+import com.mercadopago.MercadoPagoConfig;
+import com.mercadopago.client.order.OrderClient;
+import com.mercadopago.core.MPRequestOptions;
+import com.mercadopago.exceptions.MPApiException;
+import com.mercadopago.exceptions.MPException;
+import com.mercadopago.resources.order.Order;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ProcessOrderById {
+
+    public static void main(String[] args) {
+        MercadoPagoConfig.setAccessToken("{{ACCESS_TOKEN}}");
+
+        OrderClient client = new OrderClient();
+
+        Map<String, String> headers = new HashMap<>();
+        headers.put("X-Sandbox", "true");
+        MPRequestOptions requestOptions = MPRequestOptions.builder()
+                .customHeaders(headers)
+                .build();
```

Comment:
Separar as instruções para facilitar a leitura dos passos do código
```suggestion
        Map<String, String> headers = new HashMap<>();
        headers.put("X-Sandbox", "true");
        
        MPRequestOptions requestOptions = MPRequestOptions.builder()
                .customHeaders(headers)
                .build();
```